dashboard/README.md

```diff
@@ -0,0 +1,12 @@
+# StarWatch Dashboard
+This folder contains the scripts to run and dockerise the dashboard for the StarWatch pipeline project. The pipeline was created using `streamlit` and queries data from a postgres database using `psycopg2`.
+
+## Design
+![ERD](../assets/starwatch_architecture_diagram.png)
```

Comment:
Good link to the ERD diagram, to showing what data the dashboard is pulling from.

Reply:
You could also link in the streamlit and psycopg2 libraries here